components/ai_chat/resources/page/stories/components_panel.tsx

```diff
@@ -796,6 +800,8 @@ function StoryContext(props: React.PropsWithChildren<{ args: CustomArgs, setArgs
     canShowPremiumPrompt: options.args.canShowPremiumPrompt,
     isMobile: options.args.isMobile,
     isHistoryFeatureEnabled: options.args.isHistoryEnabled,
+    isContentAgentFeatureEnabled: options.args.isContentAgentFeatureEnabled,
```

Comment:
please keep consistent naming across languages as well. It's confusing and difficult to search for things when the naming is different

Reply:
renamed